administrators.rst

```diff
@@ -882,14 +882,83 @@ Replicator
 ^^^^^^^^^^
 
 Replicator locations can be configured to replicate the AIPs in one or more AIP
-storage locations. If you wish for the replicated AIPs to be encrypted, create
-the location in an :ref:`encrypted space <gpg>`.
+storage locations. Replicators can be configured for each of the following
+protocols supported by the storage service:
+
+* Local filesystem, including GPG encrypted.
+* S3.
+* Arkivum.
+* Duracloud.
+* DSpace via SWORD2 API.
+* DSpace via REST API.
+* Fedora via Sword2
+* LOCKSS-o-matic.
+* NFS.
+* Pipeline local filesystem.
+* Swift.
+
+Replicators are associated with an AIP storage location via the storage
+location's configuration page. As such Replicators must be configured before
+they can be selected for use alongside an AIP store. There is additional
+information about this under :ref:`configure location <configure_location>`. If
+you wish for the replicated AIPs to be encrypted, create the location in an
+:ref:`encrypted space <gpg>`.
 
 .. note::
 
    If you want the same space to have multiple purposes, multiple locations with
    different purposes can be created.
 
+Outcomes of a Replicator
+""""""""""""""""""""""""
+
+**One AIP to many replicators:**
+
+An AIP will be replicated *n-* times where there are *n-* configured
+Replicators for an AIP storage location.
+
+**On reingest replicas are replaced and given new identifiers:**
+
+During reingest, previous replicas will be deleted and marked as DELETED within
+the storage service Packages tab. A new replica will be created and marked as
+UPLOADED. The new replica will be assigned a new UUID. Pointer files will be
+updated with accurate references between the source and the replica and vice
+versa.
+
+**Reingest can be used to update or delete replicas:**
+
+If a Replicator is disabled then reingest can be used to selectively remove
```

Comment:
Change "If a Replicator is disabled then reingest can be used" to "If a Replicator is disabled, reingest can be used"